docs/rfc/rfc-1.adoc

```diff
@@ -435,13 +435,49 @@ As time passes and operators drop out of the system, a wallet becomes at risk
 of being able to meet the 51-of-100 threshold to produce signatures.
 Additionally, we want to avoid situations where operators are the custodians of
 a wallet for extended periods. To avoid these issues, once a wallet is older
-than the `wallet_max_age`, or if it drops below the liveness threshold (say, below 70 on
-a <<heartbeat,heartbeat>>), we motion to transfer the funds to another randomly
-selected wallet.
+than the `wallet_max_age`, or if it drops below the liveness threshold (say,
+below 70 on a <<heartbeat,heartbeat>>), we motion to
+<<closing-a-wallet,transfer the funds>> to another randomly selected wallet.
 
 Once a wallet no longer has funds and is not the primary wallet for new
-deposits, it can be closed and operators are no longer required to maintain
-it.
+deposits, it can be <<closing-a-wallet,closed>> and operators are no longer
+required to maintain it.
+
+[[closing-a-wallet]]
+==== Closing A Wallet
+Governable Parameters:
+
+- `wallet_min_btc`: The smallest amount of BTC a wallet can hold before we
+  attempt to close the wallet and transfer the funds to a randomly selected
+  wallet.
+
+When a wallet fails a <<heartbeat,heartbeat>> or ends a
+<<redemption,redemption>> with less than `wallet_min_btc` remaining, the
+operators begin the process of closing it. If there are still funds remaining,
+the <<operator-only,first operator>> posts a <<operator-only,reimbursable>>
+transaction to the ethereum chain declaring an intention to close the wallet.
+
+That operator proposes a BTC fee (as in <<sweeping,sweeping>>) and an ethereum
+block height commitment, and they use that to construct a P2PKH transaction
+moving the wallet's main UTXO to another randomishly chosen wallet where the
+source of randomness comes from the ethereum block hash at the pre-commited
+block height.
+
+After this transaction is complete, <<public-knowledge,the public>> is able
+to submit a SPV proof to let ethereum know that the funds were transferred
+nonfraudulently, and which account balances need to be updated (like the
+treasur which pays the bitcoin mining fee).
+
+*Notes*: We trust the honest majority to pick a new random wallet. Transferring
+the BTC to any address other than the P2PKH of one of the other wallet
+addresses is fraud, and is <<Punishment,punishable>>. Moving funds without
+sending a follow-up proof is <<Punishment,punishable>>. Failure to close a
+wallet that failed a <<heartbeat,heartbeat>> or fell below `wallet_min_btc` is
```

Comment:
```suggestion
wallet that failed a <<heartbeat,heartbeat>>, fell below `wallet_min_btc`, or exceeded `wallet_max_age` is
```

Reply:
addressed in [0704bae](https://github.com/keep-network/tbtc-v2/pull/127/commits/0704baebb3cdf5e953a6e6c4f41f4de939a80a3c)